roles/ovirt-mac-pools/tasks/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: set mac pool
+  ovirt_mac_pools:
+    auth: "{{ ovirt_auth }}"
+    name: "{{ mac_pool_name | default('Default') }}"
```

Comment:
it is the mac_pool found on the engine, i guess this is the one we wwant to set if mac_pool_name was not specified. 

what do u suggest?